### DISCUSSION TITLE
blank page

### DISCUSSION THREAD

**Discussion by therock247uk** [PrivateBin/PrivateBin#1440] [Q&A]
Hi trying to setup private bin on a cpanel server all I get is the following?

Internal Server Error

The server encountered an internal error or misconfiguration and was unable to complete your request.

any help would be appricated

**Comment by elrido** [comment]
This can be anything, from your environment using an unsupported PHP version to a permission error. You must check the servers logs to find out what goes wrong and if privatebin files are even executed or not. If it even goes into the application, you should find exception messages that should hint where it goes wrong.
